extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1346,6 +1346,11 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
   getA4aAnalyticsConfig() {
     return getCsiAmpAnalyticsConfig();
   }
+
+  /** @override */
```

Comment:
optional: add a comment explaining why we are forcing out of fast fetch for fluid

Reply:
Done.